### ISSUE TITLE
Need help setting up a sensor for unknown face detection

### ISSUE THREAD

**Issue by prosenba1** [sdavides/AqaraPOST-Homeassistant#8]
Sorry if it is not the place for asking, but I would like to set up an unknown face detection sensor using yaml (not nodered), so every time the camera detects an unknown face the sensor in HA reflects this situation.   Is it possible? I have Aqara g3 camera. Thanks in advance

**Comment by sdavides** [comment]
yes, it can be done, but I stopped developing the rest method, use nodered which has more functions, I don't see the reason not to use it

**Comment by prosenba1** [comment]
Thank you very much for your answer. Where may I find or try to find how to call Aqara API to know the last face detected and if it's known or unknown?. As I can see all the endpoints are different depending on what info you are trying to collect. app/v1.0/lumi/devex/face/info?did=lumi1xxxx is only bringing information about what faces it has, but not last time they were detected. Perhaps is the same endpoint but different parameters?

**Comment by sdavides** [comment]
from the logs, use burp or zap as in the pdf guide and you will find it.

**Comment by prosenba1** [comment]
Excelent! Thank you very much!
